extensions/wikia/TwitterTag/TwitterTagHooks.class.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+class TwitterTagHooks {
+	
+	/*
+	 * onParserFirstCallInit
+	 *
+	 * Registers the <twitter> tag with the parser and sets its callback
+	 *
+	 * @param $parser - The parser
+	 * @return true
+	 */
+	static public function onParserFirstCallInit( Parser &$parser ) {
```

Comment:
Objects are effectively passed by reference in PHP anyway, so you don't need the `&`.